configuration/white_labels/base.py

```diff
@@ -24,49 +24,67 @@ def get_white_label(self) -> WhiteLabel:
 
     acute_condition_options = {
         "food": {
-            "icon": {"_name": "Food", "_classname": "option-card-icon"},
-            "_label": "acuteConditionOptions.food",
-            "_default_message": "Food or groceries",
+            "icon": {"_icon": "Food", "_classname": "option-card-icon"},
+            "text": {
+                "_label": "acuteConditionOptions.food",
+                "_default_message": "Food or groceries",
+            },
```

Comment:
Just checking - updating the shape of this config doesn't have any downstream impacts?